test/data_handler/test_data_silo.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+import numpy as np
+
+from farm.modeling.tokenization import Tokenizer
+from farm.data_handler.processor import TextClassificationProcessor
+from farm.data_handler.data_silo import DataSilo, DataSiloForNestedCrossVal
+
+
+def test_data_silo_for_nested_cross_val():
+    lang_model = "bert-base-german-cased"
+    n_outer_splits = 7
```

Comment:
lets use less splits, otherwise processing takes longer than necessary. Or is there a special reason for these numbers?

Reply:
yes - done